scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh

```diff
@@ -366,7 +366,9 @@ cd ${PLOTDIR}
 tar -cvf evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar *png
 
 if [ $SENDCOM = "YES" ]; then
- cpreq -v  evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
+ if [ -s evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar ]; then
+  cp evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
```

Comment:
@PerryShafran-NOAA Can you please make your new "cp" lines become "cp -v"? Thank you!

Reply:
Done.